### PR TITLE
cleaning up examples

### PR DESCRIPTION
**Pull request by kineticsquid**

### CODE REVIEW

File: soe/python/manifest.yml [hunk 1]
```diff
@@ -13,6 +13,6 @@ applications:
     CONVERSATION_PASSWORD: #CONVERSATION PASSWORD
     CONVERSATION_WORKSPACE_ID: #WORKSPACE ID
     CONVERSATION_VERSION: '2016-07-11'
-# URL for cloudant should look like https://3f99fb1a-f7b9-4007-929d-3d4dcd810d0b-bluemix:2cb3a3b43950fd9c28d3bca97a1ed052456b75794363caced80f566ee43fc582@3f99fb1a-f7b9-4007-929d-3d4dcd810d0b-bluemix.cloudant.com
+# URL for cloudant should look like https://{credentials}:2cb3a3b43950fd9c28d3bca97a1ed052456b75794363caced80f566ee43fc582@3f99fb1a-f7b9-4007-929d-3d4dcd810d0b-bluemix.cloudant.com
```
Comment (jfmartinez) [inline soe/python/manifest.yml:16]: Might be better for it to be 
```
 # URL for cloudant should look like https://{cloudant_account}:{password}@{cloudant_account}-bluemix.cloudant.com 
```
"2cb3a3b43950fd9c28d3bca97a1ed052456b75794363caced80f566ee43fc582" is still some kind of password

and `3f99fb1a-f7b9-4007-929d-3d4dcd810d0b` in the url is also the cloudant username

File: voice-agent-openid-provider/config/registry/registry-incl.xml [hunk 2]
```diff
@@ -13,7 +13,7 @@
 <server>
    <basicRegistry id="basic" realm="OpBasicRealm">
       <!-- replace / add user -->
-      <user name="bpulito" password="{hash}ATAAAAAI7tXy93T2DDdAAAAAIEny+m0RVHlDl2o+hkRcpihcpogPTuI6QznzUMHzYmKd" />
+      <user name="username" password="{password hash" />
```
Comment (jfmartinez) [inline voice-agent-openid-provider/config/registry/registry-incl.xml:16]: Weird hash replacement happened in this file, shouldn't this be 

```
password="{hash} hashed_password" 
```

File: soe/python/manifest.yml (hunk 1 above)
Comment (kineticsquid) [reply soe/python/manifest.yml]: Thanks @jfmartinez, I missed that.

File: soe/python/startup.sh [hunk 3]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-export CLOUDANT_URL="https://cb3fca4e-e0f6-4a7b-9a98-f3c8ed9d4820-bluemix:e7833b543725775f89ff1f0915c1f67135423a2d5b7746588ce99c2f9b72dc06@cb3fca4e-e0f6-4a7b-9a98-f3c8ed9d4820-bluemix.cloudant.com"
-export CONVERSATION_PASSWORD="0oSVoDfHiQCl"
-export CONVERSATION_USERNAME="9ff51c5e-5cb2-4877-b7ca-a7f3194ca8ba"
+export CLOUDANT_URL="https://{credentials}:e7833b543725775f89ff1f0915c1f67135423a2d5b7746588ce99c2f9b72dc06@cb3fca4e-e0f6-4a7b-9a98-f3c8ed9d4820-bluemix.cloudant.com"
```
[the remark this answers is not in the document]
Comment (kineticsquid) [reply soe/python/startup.sh]: Fixed here also

File: voice-agent-openid-provider/config/registry/registry-incl.xml (hunk 2 above)
Comment (kineticsquid) [reply voice-agent-openid-provider/config/registry/registry-incl.xml]: Not sure what happened with that edit. Thanks.

**Comment by kineticsquid** [comment]
@jfmartinez How's this?

**Comment by kineticsquid** [comment]
Thanks @jfmartinez

**Pull request merged**